.github/CONTRIBUTING.md

```
@@ -73,27 +124,45 @@ If the solution requires more time, then decompose it into smaller independent P
 
 We usually reject and close PRs which do not have activity for the last 24 hours, unless there is a clear comment explaining the reason why that PR is stalled.
 
+## CI Checks
+
+To maintain the quality and integrity of our project, all PRs must successfully pass the required Continuous Integration (CI) checks before being marked as "ready to review." PRs with failing CI checks will be rejected.
+
+The required checks are as follows:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)